pygmt/src/meca.py

```diff
@@ -424,7 +424,8 @@ def meca(  # noqa: PLR0912, PLR0913, PLR0915
             )
     elif isinstance(spec, np.ndarray):  # spec is a numpy array
         if convention is None:
-            raise GMTInvalidInput("'convention' must be specified for an array input.")
+            msg = "'convention' must be specified for an array input."
```

Comment:
```suggestion
            msg = "'Convention' must be specified for an array input."
```

Reply:
It should be lower case since it's a parameter name in quotes.